.gitignore

```diff
@@ -25,3 +25,5 @@ bazel-*
 .bazelproject
 .aswb
 *.pb
+gradle.properties
+gradle-wrapper.properties
```

Comment:
Oh, Thank you very much. Ok, I will revert it immediately

Reply:
I have reverted it @adhiamboperes

Reply:
It still shows 2 files changed. I'm only expecting to see app's build.gradle modified.

Reply:
ok, checking

Reply:
Hi @adhiamboperes, I have tried to revert the change, but i am still having two file changes as you said. Even used rebase. I might have to advise to use a new commit branch, if you agree.

Reply:
@DeonWaju, it may be best to open a replacement PR. When creating a commit, only git add the file that has the desired changes, instead of git add .. This should be simple to do from within the android studio commit toolbar/vcs.

Reply:
Ok, I understand, thanks very much